docs/shared/recipes/add-stack/add-nuxt.md

```diff
@@ -0,0 +1,139 @@
+
+## Create Nuxt App
+
+Using `nuxi init`, `cd` to the directory above where you want the app folder to live and run
+
+```shell
+❯ npx nuxi@latest init <app-name>
+Nuxi 3.6.5
+✨ Nuxt project is created with v3 template.
+```
+
+Tell Nx how to cache the build with the `nx` key in the `package.json`.
+You'll also want to move the dependencies/devDependencies from the project level `package.json` to the top level `package.json` to help maintain a [single version policy](/concepts/more-concepts/dependency-management#single-version-policy).
+
+```json {% fileName="apps/<app-name>/package.json" %}
+{
+  "name": "<app-name>",
+  "scripts": {/***/}
+  "nx": {
+    "targets": {
+      "build": {
+        "outputs": ["{projectRoot}/.output", "{projectRoot}/.nuxt"]
+      }
+    }
+  }
+}
+```
+
+Make sure to ignore the `.output` and `.nuxt` folders from git and the Nx cache inputs.
+
+```json {% fileName="nx.json %}
+{
+  "namedInputs": {
+    "default": [
+      "{projectRoot}/**/*",
+      "!{projectRoot}/.nuxt/**/*", // ignore nuxt specific output folders
+      "!{projectRoot}/.output/**/*",
+      "sharedGlobals"
+    ],
+    "production": ["default", "!{projectRoot}/.eslintrc.json"],
+    "sharedGlobals": []
+  }
+}
+```
+
+### Project Aliases
+
+Tell Nuxt how to use existing TS Paths defined in the `tsconfig.base.json` via the `alias` field in the `nuxt.config.ts` file.
+
+> Nuxt [generates a tsconfig](https://nuxt.com/docs/guide/directory-structure/tsconfig) with these aliases as TS Paths
+
+```ts {% fileName="apps/<app-name>/nuxt.config.ts" %}
+import { defineNuxtConfig } from 'nuxt/config';
+import { join } from 'path';
+import { workspaceRoot } from '@nx/devkit';
+
+function getMonorepoTsConfigPaths(tsConfigPath: string) {
+  const tsPaths = require(tsConfigPath)?.compilerOptions?.paths as Record<
+    string,
+    string[]
+  >;
+
+  const alias: Record<string, string> = {};
+  if (tsPaths) {
+    for (const p in tsPaths) {
+      alias[p.replace(/\/\*$/, '')] = join(
+        workspaceRoot,
+        tsPaths[p][0].replace(/\/\*$/, '')
+      );
+    }
+  }
+
+  return alias;
+}
+
+export default defineNuxtConfig({
+  alias: getMonorepoTsConfigPaths('../../tsconfig.base.json'),
+  devtools: { enabled: true },
+});
+```
+
+### Setup Project Graph
+
+You can use [`tags`](https://nx.dev/reference/project-configuration#tags) with [`implicitDependencies`](https://nx.dev/reference/project-configuration#implicitdependencies) to define links between projects in the project graph.
+
+```json {% fileName="apps/<app-name>/package.json %}
+
+{
+  "name": "<app-name>",
+  "scripts": {/***/}
+  "nx": {
+    "tags": ["scope:<app-name>"],
+    "implicitDependencies": ["tag:scope:<app-name>"]
+  }
+}
+```
+
+and in the projects that are imported via `.vue` files can add the `scope:<app-name>` tag in the `project.json`:
+
+```json
+{
+  "tags": ["scope:<app-name>"]
+}
+```
```

Comment:
I don't know what you're telling people to do here.

Reply:
this is building the graph via the tags and implicit dependencies, it's a second part to the stuff above this.